script/lint.js

```diff
@@ -43,7 +43,7 @@ function spawnAndCheckExitCode (cmd, args, opts) {
 
 const LINTERS = [ {
   key: 'c++',
-  roots: ['atom'],
+  roots: ['atom', 'native_mate'],
   test: filename => filename.endsWith('.cc') || filename.endsWith('.h'),
```

Comment:
`clang-format` can handle `.mm` files, lets try adding it.

Reply:
however `cpplint.py` does not like it
```
Ignoring atom/app/atom_library_main.mm; not a valid file name (cc, h, cpp, cu, cuh)
etc.
```

Reply:
added run clang-format on Objective-C files, but not cpplint.py

Reply:
actually, I will do that in another PR. I can enable limited `cpplint.py`

Reply:
#18104